test/unit/ngsiv2/general/deviceService-test.js

```diff
@@ -193,12 +221,42 @@ describe('NGSI-v2 - Device Service: utils', function () {
                 .post('/v2/entities?options=upsert')
                 .reply(204);
 
-            async.series(
-                [utils.request.bind(utils.request, groupCreation), utils.request.bind(utils.request, deviceCreation)],
-                function (error, results) {
-                    done();
-                }
-            );
+            async.series([request.bind(request, groupCreation), request.bind(request, deviceCreation)], function (
+                error,
+                results
+            ) {
+                done();
+            });
+        });
```

Comment:
Those changes were not included in PR #954. Could you elaborate why are included here?